pkg/cloud/services/ec2/ami.go

```diff
@@ -56,8 +56,8 @@ const (
 	// 1. the string value `capa-ami-`
 	// 2. the baseOS of the AMI, for example: ubuntu-18.04, centos-7, amazon-2
 	// 3. the kubernetes version as defined by the packages produced by kubernetes/release with or without v as a prefix, for example: 1.13.0, 1.12.5-mybuild.1, v1.17.3
-	// 4. a `-` followed by any additional characters.
-	DefaultAmiNameFormat = "capa-ami-{{.BaseOS}}-?{{.K8sVersion}}-*"
+	// 4. followed by any characters.
+	DefaultAmiNameFormat = "capa-ami-{{.BaseOS}}-?{{.K8sVersion}}*"
```

Comment:
> From the recent releases, the new AMIs are getting generated as `capa-ami-amazon-2-v1.25.3-1665727783`

I think the old pattern should match the above AMI, if `BaseOS` is `amazon-2`, and `K8sVersion` is `v1.25.3`.

I must be misunderstanding something...

Reply:
I think I get what you are saying. But not sure why this format is not working while populating the AMIs to S3. And the only diff I see in the naming is -00 suffix .

Reply:
Found the culprit, and updated the PR and description.